samples/snippets/requirements.txt

```diff
@@ -3,10 +3,10 @@ google-cloud-bigquery-storage==2.9.0
 google-auth-oauthlib==0.4.6
 grpcio==1.41.0
 ipython==7.16.1; python_version < '3.7'
-ipython==7.17.0; python_version >= '3.7'
+ipython==7.29.0; python_version >= '3.7'
 matplotlib==3.3.4; python_version < '3.7'
 matplotlib==3.4.1; python_version >= '3.7'
 pandas==1.1.5; python_version < '3.7'
-pandas==1.3.2; python_version >= '3.7'
-pyarrow==5.0.0
+pandas==1.3.4; python_version >= '3.7'
```

Comment:
Hm... I noticed that in `constraints-3.6` we pin `pandas` to `0.24.2`, and `setup.py` pins it to as low as `pandas>=0.23.0`.

I forgot if we intentionally only test samples with up-to-date dependencies?

Reply:
For samples we try to use the latest working version of everything, but that's a good point re: setup.py. We likely forgot to update that when updating the constraints to fix some test.